src/read_vegparam.c

```diff
@@ -239,16 +239,23 @@ veg_con_struct *read_vegparam(FILE *vegparam,
         nrerror(ErrStr);
       }
 
+      for ( j = 0; j < 12; j++ ) {
```

Comment:
OK, I see, the changes that get merged into 5.0 should ideally look like 5.0 in terms of formatting.